Pocket/pocket-ios
# SwiftUI Home - Add remaining card actions

## Goal
This PR adds the remaining card actions to home. Specifically
* Favorite/unfaforite
* Overflow:
   - Archive (a saved item)
   - Delete (a saved item)
   - Share
   - Report

## Test Steps
* Build/run and verify the above actions are available where appropriate and produce the expected result.
